Shared/Features/Freaks/ViewModel/FreaksViewModel.swift

```diff
@@ -7,6 +7,10 @@ final class FreaksViewModel: ObservableObject {
     @Published var skillSelections: [Skill] = []
     @Published var shouldFilterFreaks: Bool = false
 
+    var counter: Int {
+        skillSelections.count
+    }
+
```

Comment:
I think we should rename this to skillsCounter

Reply:
Maybe not, let's keep it generic to account for the generic Filter View

Reply:
Maybe rename it to something else, but not skillsCounter